social-tw/social-tw-website
# [Test] Login Frontend

## Summary

Completed the setup of the Testing Library and merged the Playwright settings branch. Added several DOM test cases to enhance our testing coverage.

## Linked Issue

#36 

## Details

- Set up the Testing Library for frontend testing.
- Merged the Playwright settings branch.
- Introduced new DOM test cases.

## Impacted Areas

- Frontend Testing setup
- Playwright settings

## Tests

- Add new DOM test cases.
- Testing was performed on the local development environment.
- All tests ran successfully without issues.

## Possible Impacts

The changes made primarily focus on testing, so there shouldn't be any impact on the actual functionality of the application. However, there could be potential overlaps or conflicts with other testing tools or configurations, if any.

## Verification Steps

1. Pull the latest changes from this branch.
2. Run the test suite using the command `yarn test`, `yarn test:e2e`, `yarn test:dom`.
3. Ensure all tests pass without any errors.
4. Review the new DOM test cases to ensure they cover the desired functionality.

## Todo

- [ ] Add more DOM test cases.
- [ ] Add new Playwright test case.

## Checklist
